### ISSUE TITLE
raphnet wiitar is not being detected

### ISSUE THREAD

**Issue by wooperworks** [InvoxiPlayGames/FestivalInstrumentMapper#19]
it works in clone hero and joy2key, but cant get the mapper to detect it

**Comment by sanjay900** [comment]
This will be fixed in the next release, have just added the missing devices.

**Comment by wooperworks** [comment]
> This will be fixed in the next release, have just added the missing devices.

awesome, thanks!

**Comment by renatus-lupus** [comment]
Make sure your adapter's driver is updated with Raphnet's utility, I ran into that issues. Now my issue is, while the program says the guitar is mapped, it does nothing in Fortnite

**Comment by sanjay900** [comment]
Yeah thats what the fix does, we were missing the ids for the other raphnet firmwares but once a fix is cut that will allow them to work.
